test/integration_tests/test_similarity.py

```diff
@@ -218,7 +218,7 @@ def test_should_do_faiss_index_scan(self):
         # Execution with index scan.
         create_index_query = """CREATE INDEX testFaissIndexScanRewrite1
                                     ON testSimilarityFeatureTable (feature_col)
-                                    USING HNSW;"""
+                                    USING FAISS;"""
```

Comment:
Can we show the query plans before and after index in the PR notes? Maybe also in the code as a comment for reference